src/ioq/src/ioq.erl

```diff
@@ -50,6 +51,9 @@ call(Fd, Msg, Metadata) ->
             queued_call(Fd, Msg, Priority)
     end.
 
+get_queue_lengths() ->
+    gen_server:call(?MODULE, get_queue_lengths).
```

Comment:
Ooh, I'd love to see this in `_stats` or `_system` output...I don't believe we have this today, do we?

Reply:
Not sure, but I had to modify this IOQ lib here because `weatherreport` previously got [something like this](https://github.com/apache/couchdb-ioq/blob/main/src/ioq.erl#L56) from the old Cloudant IOQ library, and this seemed like the closest analogy to that metric. You can see later in this file that there's a compatibility clause that depends on the result of `erlang:function_exported(ioq, get_queue_lengths, 0)`.  But this is basically a newly exposed stat for this IOQ lib, and I have no idea how useful it might end up being in practice.

Reply:
Given we report on all the message queues in `_system`, we should maybe expose this for people just in case it tells them anything useful. Happy to put that into another PR though. 

@kocolosk since you took the IOQ2 work over the finish line for CouchDB 3.x, what do you think?